extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -21,6 +21,8 @@ let dacfxService: mssql.IDacFxService;
 let schemaCompareTester: SchemaCompareTester;
 const dacpac1: string = path.join(__dirname, '../testData/Database1.dacpac');
 const dacpac2: string = path.join(__dirname, '../testData/Database2.dacpac');
+const includeExcludeSourceDacpac: string = path.join(__dirname, '../testData/SchemaCompareIncludeExcludeSource.dacpac');
```

Comment:
The path parts should be passed in separately (`path.join(__dirname, '..', 'testData', 'SchemaCompareIncludeExcludeSource.dacpac')`)